recipe/meta.yaml

```diff
@@ -317,6 +329,7 @@ outputs:
         - pip check
         - python -c "import torch; print(torch.__version__)"  # [not (aarch64 and cuda_compiler_version != "None")]
         - python -c "import torch; assert torch.backends.mkldnn.m.is_available()"  # [x86 and cuda_compiler_version == "None"]
+        - python -c "import torch; torch.tensor(1).numpy(); print('numpy support enabled!!!')"
```

Comment:
Should this be changed to an assert as well to avoid future issues?

Reply:
Torchani was failing with:

```
2024-08-23T23:03:51.2530690Z         self.results['free_energy'] = energy.item()
2024-08-23T23:03:51.2531050Z     
2024-08-23T23:03:51.2531550Z         if 'forces' in properties:
2024-08-23T23:03:51.2532540Z             forces = -torch.autograd.grad(energy.squeeze(), coordinates, retain_graph='stress' in properties)[0]
2024-08-23T23:03:51.2533270Z >           self.results['forces'] = forces.squeeze(0).to('cpu').numpy()
2024-08-23T23:03:51.2533820Z E           RuntimeError: PyTorch was compiled without NumPy support
2024-08-23T23:03:51.2534420Z E           
2024-08-23T23:03:51.2534900Z E           To execute this test, run the following from the base repo dir:
2024-08-23T23:03:51.2535460Z E                python test_ase.py -k TestASE.testWithNumericalForceWithPBCEnabled
2024-08-23T23:03:51.2535950Z E           
2024-08-23T23:03:51.2536240Z E           This message can be suppressed by setting PYTORCH_PRINT_REPRO_ON_FAILURE=0
2024-08-23T23:03:51.2536620Z 
2024-08-23T23:03:51.2537390Z ../../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.9/site-packages/torchani/ase.py:92: RuntimeError
```

https://github.com/conda-forge/torchani-feedstock/pull/39

Reply:
I updated the syntax of the test (just now) to follow the failed case of torchani